lib/types/Color3.lua

```diff
@@ -1,11 +1,21 @@
 local Color3 = {}
 
-function Color3.new(r, g, b)
-	return {
-		r = r,
-		g = g,
-		b = b,
-	}
+function Color3.new(...)
+	if select("#", ...) == 0 then
+		return Color3.new(0, 0, 0)
+	else
+		local r, g, b = ...
+
+		return {
+			r = r,
+			g = g,
+			b = b
```

Comment:
Can you add a trailing comma here like the original block? It isn't that important here, but it makes diffs cleaner and makes it easier to change line ordering.